### PR TITLE
Export SVG icons names

### PR DESCRIPTION
**Pull request by 3imed-jaberi**
Create a script to generate SVG icons names based on the "svg" directory which helps us to put all the icons inside the new design system storybook ([Iconghraphy](https://paacklogistics.atlassian.net/browse/DES-137)).

### CODE REVIEW

File: src/utils/getSVGFilename.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const { join } = require('path');
+const { readdirSync, writeFileSync } = require('fs');
+
+(function getSVGFilenames () {
+  const basePath = join(process.cwd(), 'src')
+  const dir = readdirSync(join(basePath, 'svg'))
+  const filenames = dir.map(file => file.split('.')[0])
+  writeFileSync(join(basePath, 'js', './SVGIconNames.json'), JSON.stringify(filenames), 'utf-8')
+})()
```
Comment (PedroHLC) [inline src/utils/getSVGFilename.js:9]: any special reason for creating a function, giving it a name, to anonymously invoke her and leave?

File: package.json [hunk 2]
```diff
@@ -6,8 +6,10 @@
     "js"
   ],
   "scripts": {
-    "prepublish": "yarn run build",
+    "prepublish": "yarn run gen:names && yarn run build",
+    "gen:names": "node src/utils/getSVGFilename.js",
     "build": "yarn run build:spritesheet && yarn run build:parcel",
+    "postbuild": "cp src/js/SVGIconNames.json js/SVGIconNames.json",
     "build:parcel": "parcel build src/js/*.js --out-dir ./js",
```
Comment (PedroHLC) [inline package.json:13]: if you guys want to replace Parcel with any other **bundler**, feel free to open a PR.

Comment (3imed-jaberi) [reply package.json:13]: we don't need to replace the bundler, just we need to copy the generated names to the published directory and in our case, it's the `js` dir.

File: src/utils/getSVGFilename.js (hunk 1 above)
Comment (3imed-jaberi) [reply src/utils/getSVGFilename.js]: no special reason just to describe what the function does... otherwise, I remove the IIFE.

File: .gitignore [hunk 3]
```diff
@@ -1,6 +1,7 @@
 .DS_Store
 .env
 
+src/js/*.json
```
Comment (mtzfactory) [inline .gitignore:4]: Why the file with svg name is being ignored?
[It won't get published inside the package](https://docs.npmjs.com/cli/v8/using-npm/developers#keeping-files-out-of-your-package) as there is no `.npmigonore` file

**Review by mtzfactory — CHANGES_REQUESTED** [review]

**Review by mtzfactory — APPROVED** [review]

**Pull request merged**